scripts/wasm-rebuild/docker-scripts/genbytecode.sh

```diff
@@ -93,6 +93,6 @@ for (var optimize of [false, true])
 }
 EOF
     chmod +x solc
-    ./solc *.sol > /tmp/report.txt
+    ./solc -- *.sol > /tmp/report.txt
```

Comment:
Why --?

Reply:
If you have a file that's named like a command line option, `*` will match it. For example if there's a file called `-rf` and you do `rm *`, you're actually doing `rm -rf *` :) To avoid this the convention in many programs is that stuff after `--` is never interpreted as an option. So `rm -- *` is safe even if you have a file called `-rf`.

Another way to avoid that is to use `./*` instead of `*` because for most programs the difference between `x.txt` and `./x.txt` does not matter. Unfortunately for `solc` it does (though it might change soon: #4702). I used `./*` in most places that shellcheck pointed out becuase that's more universal but I could not do it here. Fortunately `boost::program_options` does support `--` so that's the workaround I used.

Reply:
Ah ok!